Unmet peer dependency insert-css@^0.2.0

Hi @yoshuawuyts!

`npm install sheetify`

gives

```sh
+-- UNMET PEER DEPENDENCY insert-css@^0.2.0
`-- sheetify@5.0.0
  +-- falafel@1.2.0
  | `-- isarray@0.0.1
  `-- through2@2.0.1
    `-- readable-stream@2.0.6
      `-- isarray@1.0.0
```

That would not be bad but running browserify gives `Cannot find module 'insert-css' from 'C:\Users\dmitry\Dropbox\Projects\gl-spectrum'`.

Why `inner-css` dependency is considered peer, when it is critical to run sheetify?

--- a/package.json
+++ b/package.json
@@ -22,6 +22,7 @@
   ],
   "dependencies": {
     "falafel": "^1.2.0",
+    "insert-css": "^0.2.0",
     "is-stream": "^1.0.1",
     "map-limit": "0.0.1",
     "postcss": "^5.0.10",

